docs/source/app-dev/index.rst

```diff
@@ -0,0 +1,80 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Writing applications using the Ledger API
+#########################################
+
+.. toctree::
+   :hidden:
+
+   services
+   daml-lf-translation
+
+DAML contracts are stored on a ledger. In order to exercise choices on those contracts, create new ones, or read from the ledger, you need to use the **Ledger API**. (Every ledger that DAML can run on exposes this same API.) And to do anything sophisticated, you'll want to write an application. 
+
+Resources available to you
+**************************
+
+- **The Java bindings**: a library to help you write idiomatic applications using the Ledger API in Java.
+
+  :doc:`Read the documentation for the Java bindings </app-dev/bindings-java/index>`
+- **The experimental node.js bindings**: a library to help you write idiomatic applications using the Ledger API in JavaScript.
+
+  :doc:`Read the documentation for the node.js </app-dev/bindings-js/index>`
+- **The underlying gRPC API**: if you want to write an application for the ledger API in other languages, you'll need to use `gRPC <https://grpc.io>`__ directly.
+
+  :doc:`Read the documentation for the gRPC API </app-dev/grpc/index>`
+- **The application architecture guide**: this documentation gives high-level guidance on how to build your application.
+
+  :doc:`Read the application architecture guide </app-dev/app-arch>`
+
+What's in the Ledger API
+************************
+
+No matter how you're accessing it (Java bindings, node.js bindings, or gRPC), the Ledger API exposes the same services:
+
+- Submitting commands to the ledger
+
+  - Use the **command submission service** to submit commands (create a contract or exercise a choice) to the ledger.
+  - Use the **command completion service** to track the status of submitted commands.
+  - Use the **command service** for a convenient service that wraps both of the above.
+- Reading from the ledger
+
+  - Use the **transaction service** to retrieve transactions of events (contracts created and contracts archived) from the ledger.
+  - Use the **active contract service** to quickly bootstrap an application with active contracts. It means you don't need to read from the beginning of the ledger or process create events for contracts that have already been archived.
+- Utility services
+
+  - Use the **package service** to query the DAML packages deployed to the ledger.
+  - Use the **ledger identity service** to retrieve the Ledger ID of the ledger the application is connected to.
+  - Use the **ledger configuration service** to retrieve some dynamic properties of the ledger, like minimum and maximum TTL for commands.
+  - Use the **time service** to obtain the time as known by the ledger server.
+
```

Comment:
Not sure whether is makes sense to mention the reset service (which is optional and for testing -- just like the time service).